nipype/interfaces/niftyreg/base.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+
+"""
+The niftyreg module provides classes for interfacing with `niftyreg
+<http://sourceforge.net/projects/niftyreg/>`_ command line tools.
+
+These are the base tools for working with niftyreg.
+
+Registration tools are found in niftyreg/reg.py
+Every other tool is found in niftyreg/regutils.py
+
+Examples
+--------
+See the docstrings of the individual classes for examples.
+
+"""
+
+from __future__ import (print_function, division, unicode_literals,
+                        absolute_import)
+from builtins import property, super
+from distutils.version import StrictVersion
+import os
+import subprocess
+from warnings import warn
+
+from ..base import CommandLine, isdefined, traits
+from ...utils.filemanip import split_filename
+
+
+def get_custom_path(command):
+    try:
+        specific_dir = os.environ['NIFTYREGDIR']
+        command = os.path.join(specific_dir, command)
+        return command
+    except KeyError:
+        return command
+
+
+def no_niftyreg(cmd='reg_f3d'):
+    if True in [os.path.isfile(os.path.join(path, cmd)) and
+                os.access(os.path.join(path, cmd), os.X_OK)
+                for path in os.environ["PATH"].split(os.pathsep)]:
+        return False
+    return True
+
+
+class PositiveInt(traits.BaseInt):
```

Comment:
@satra, thx. I just changed all of them to use Range traits instead of rewritting something that does the same.

Cheers,